README.md

```diff
@@ -25,6 +29,34 @@ export GAZEBO_MODEL_PATH=`pwd`/models
 gazebo worlds/small_house.world
 ```
 
+## Example: Running this world on Gazebo headless and running the UI on Gzweb
+*Tested in ROS Kinetic, Gazebo 7 with node version 10.22.1*
```

Comment:
Could you include a more dire warning regarding the potential pain of installing Gzweb, outside of these tested versions?  ( I tested in ROS Melodic, Gazebo 9.14 , Node 8.11.3)

Reply:
For avoiding the pain of installation for gzweb I created a dockerfile which works with ROS kinetic, gazebo 7.16 and node version 10.22.1 the image could be easily pulled from https://hub.docker.com/r/kapildesh/gzweb7.

Reply:
Great, thank you!